### PR TITLE
Remove secrets repo

### PR DESCRIPTION
**Pull request by carmat88**
## Change content and motivation
Removing the need of using external repo for secrets from KubeNow image building process . Global environmental variables have been set in CI.

## GitHub cross-links 

**Fixes:** 

**Docs**:

### CODE REVIEW

**Comment by carmat88** [comment]
@mcapuccini Hey Marco, whenever you have time, let me know what do you think of the implemented revisions for removing the need of the secrets-kubenow repo. Many thanks!

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 language: python
 
+notifications:
```
Comment (mcapuccini) [inline .travis.yml]: please activate notifications

**Review by mcapuccini — CHANGES_REQUESTED** [review]
Only a very minor thing, then we can merge!

File: .travis.yml (hunk 1 above)
Comment (carmat88) [reply .travis.yml]: Ah, forgot to remove it! Thanks for having checked. It has been removed as requested.

**Comment by carmat88** [comment]
@mcapuccini It's done!

**Review by mcapuccini — APPROVED** [review]

**Pull request merged**